### PR TITLE
Update SendTransaction

### PR DESCRIPTION
**Pull request by JeongGoEun**
Update SendTransaction

### CODE REVIEW

File: src/components/SendTransaction/SendTransaction.react.js [hunk 1]
```diff
@@ -1,22 +1,71 @@
...
 
+var QRCode = require('qrcode.react');
+
+export default class SendTransaction extends Component {
   constructor() {
     super();
+    this.state = {
+      to: '',
+      value: '',
+      data: '',
```
Comment (hexoul) [inline src/components/SendTransaction/SendTransaction.react.js]: to, value and data is not appropriate to place in state.

**Comment by JeongGoEun** [comment]
Delete to, value and data in state

File: src/components/SendTransaction/SendTransaction.react.js [hunk 2]
```diff
@@ -1,22 +1,71 @@
...
-        SendTransaction
+        <div className={styles.test}>
+          Uri: {this.state.trxRequestUri != undefined && this.state.trxRequestUri} <br />
+          to: {this.state.to != undefined && this.state.to} <br />
+          value: {this.state.value != undefined && this.state.value} <br />
+          data: {this.state.data != undefined && this.state.data} <br />
+        </div>
+
+        <div>
+        <Popup trigger={<Button>SendTransaction</Button>}
```
Comment (hexoul) [inline src/components/SendTransaction/SendTransaction.react.js]: Popup should be rendered after initializing `this.state.trxRequestUri`

File: src/components/SendTransaction/SendTransaction.react.js [hunk 3]
```diff
@@ -1,22 +1,71 @@
...
+
+  onCloseLogin() {
+    clearInterval(this.interval);
   }
 
   render() {
     return (
       <div>
-        SendTransaction
+        <div className={styles.test}>
```
Comment (hexoul) [inline src/components/SendTransaction/SendTransaction.react.js]: Delete line50-55

**Pull request merged**